pr2eus/pr2-interface.l

```diff
@@ -324,6 +325,82 @@ Example: (send self :gripper :rarm :position) => 0.00"
 	 (when (and current origin)
 	   (mapcar #'v- current origin))
        current)))
+  (:switch-controller
+   (stop start &key (strictness 1))
```

Comment:
Please add docstring including
- method usage
- description of each argument
- description of return value
- note